firefox_puppeteer/ui/menu.py

```diff
@@ -6,10 +6,10 @@
 from marionette_driver.errors import NoSuchElementException
 
 from firefox_puppeteer import DOMElement
-from firefox_puppeteer.base import BaseLib
+from firefox_puppeteer.ui_base_lib import UIBaseLib
 
 
-class MenuBar(BaseLib):
+class MenuBar(UIBaseLib):
```

Comment:
could you please clarify what is needed to be done here? i cannot find anything more general than a menubar as a menu